src/components/PushAudienceDialog/PushAudienceDialog.react.js

```diff
@@ -91,6 +92,10 @@ export default class PushAudienceDialog extends React.Component {
   }
 
   handleAddCondition() {
+    if (!this.props.schema || !Object.keys(this.props.schema).length) {
```

Comment:
NO NEED TO CHANGE, but whenever I see something like this, I change it to:

```!(this.props.schema && Object.keys(this.props.schema).length)```

it's easier for ME to read (ymmv)